packages/suite/src/support/messages.ts

```diff
@@ -7508,9 +7508,10 @@ export default defineMessages({
         id: 'TR_LOOKING_FOR_COINJOIN_ROUND',
         defaultMessage: 'Waiting for a round',
     },
-    TR_COINJOIN_ROUNDS_LEFT: {
-        id: 'TR_COINJOIN_ROUNDS_LEFT',
-        defaultMessage: '{rounds, plural, one {# round} other {# rounds}} left',
+    TR_COINJOIN_SESSION_COUNTDOWN: {
+        id: 'TR_COINJOIN_SESSION_COUNTDOWN',
+        description: 'deadline including unit i.e. 5 hours or 15 minutes',
+        defaultMessage: '{deadline} left',
```

Comment:
What's up with the plural version? You decided not to include it?

Reply:
I have no idea how to deal with that without rewriting it completely again or creating special logic just to get the value. I'm not even sure what value to pass - the first one? i.e. `3 hodiny 1 minuta` -> pass 3 -> plural? I guess it works for czech, don't know about other languages...

Reply:
b1396847a

Reply:
see the new note in Description: 
>`TR_COINJOIN_ROUND_COUNTDOWN` replaced with `TR_COINJOIN_ROUND_COUNTDOWN_PLURAL` that receive {value} and {firstValue} instead of {time} param to enable handling plural. 
There is also new `TR_COINJOIN_SESSION_COUNTDOWN_PLURAL` message where those params are definitelly useful at least for Czech – `{firstValue, plural, one {zbývá} few {zbývají} other {zbývá}} {value}` can be used resulting to `zbývá 1 hodina / zbývají 2 hodiny / zbývá 5 hodin`...

Reply:
Yeah, this is the solution we discussed :)

Reply:
I just didn't figure out how to get to that value at first. And also it can't be applied on TR_LEFT, but I believe it's not needed there.